ssp

```diff
@@ -8826,6 +8828,22 @@ sub check_for_cpro {
 
 ## END malware checks
 
+sub detect_slowloris {
+    my $sockcount = 75;
+    my $count = 0;
+
+    return unless my $netstat_out = timed_run( 0, 'netstat', '-nap' );
```

Comment:
Please use the existing @LSOF_80 which contains the contents of "lsof -n -i 'tcp:80'" instead of running netstat.